cmd/operator/main.go

```diff
@@ -160,6 +166,12 @@ func flagsAreValid() (bool, error) {
 		}
 	}
 
+	var err error
+	logrusLevel, err = logrus.ParseLevel(flLoggingLevel)
+	if err != nil {
+		return false, errors.Wrap(err, "invalid verbosity value")
```

Comment:
this should be art of init/main CLI flag parsing. as you can imagine, it's too late to initialize a logger here.

you initialize the logger in main() so why are you doing it here?

Reply:
It could go in `main`. The only small side consequence is that it'd add an extra `Fatal` call.

Reply:
that's fine, those are early in the program, so we'd much rather fail early before we set up other things.